fili-core/src/main/java/com/yahoo/bard/webservice/data/config/provider/MetricConfiguration.java

```diff
@@ -0,0 +1,56 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.config.provider;
+
+import com.yahoo.bard.webservice.data.dimension.DimensionDictionary;
+import com.yahoo.bard.webservice.data.metric.LogicalMetric;
+import com.yahoo.bard.webservice.data.metric.MetricDictionary;
+
+/**
+ * A metric with a definition, description, and flag to exclude from external API
+ *
+ * FIXME: The isExcluded() flag is currently unused.
+ */
+public interface MetricConfiguration {
+
```

Comment:
Perhaps this class doesn't need to know about temporary dictionaries.  Perhaps load everything into a single dictionary and then filter out things that need filtering as a clean up stage.